ios/platform/README.md

```diff
@@ -1,9 +1,9 @@
-Files in this folder are copied into the iOS build directory
+Files in this folder are copied directory into the iOS build directory
 when the iOS app is compiled:
 
     <project-dir>/build/iphone
 
-You can place files such as asset catalog files and storyboards in this
+You can place files such as asset catalog files, .framework files and storyboards in this
 directory.
```

Comment:
```suggestion
folder.
```

Reply:
this is a generated message, not going to change it